models/models.js

```diff
@@ -27,29 +27,22 @@ const fetchSingleArticle = (id) => {
   })
 };
 
-// const user = rows[0];
-// if (!user) {
-//   return Promise.reject({
-//     status: 404,
-//     msg: `No user found for user_id: ${user_id}`,
-//   });
-// }
-// return user;
-
-
-
-// const fetchComments = (id) => {
-//   const queryString = `SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at ASC`;
-//   return db.query(queryString, [id]).then((result) => {
-//     return result.rows;
-//   });
-// };
-
-// GROUP BY articles.article_id ORDER BY created_at
+const fetchComments = (id) => {
+  const queryString = `SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at ASC`;
+  return db.query(queryString, [id]).then((result) => {
+    if (result.rows.length < 1) {
+      return Promise.reject({
+        status: 404,
+        msg: "Comments not found",
+      });
```

Comment:
You've got a promise.reject here but I've not actually seen the testing for it! Double check that you're definitely testing that this works please 😄